test-app/src/custom-components/homepage-template.less

```diff
@@ -13,7 +13,7 @@
 
 .HomepageTemplate-aboveFold {
   width: 100%;
-  min-height: 100vh;
+  min-height: calc(100vh - 5px);
```

Comment:
I'm just putting this comment for knowledge sharing reasons here:

I was wondering how this is not picked up by LESS and is just literally passed through to the browser. Now I learned that LESS has changed Math interpretations (that's also why it requires parenthesis for divisions but not subtractions): https://lesscss.org/usage/#less-options-math

This means that we can remove the bulky string escaping we have in older `calc` usages in our LESS files and move to this cleaner approach.

Reply:
Good point, I updated the 2 `calc` usages that composed the calc via string 👍 
And I used a well names LESS variable for the `5px` in the `min-height` calculation above to make the intention clear (me not being a comments fan 😊 ).